dataactcore/read_config.py

```diff
@@ -4,8 +4,13 @@
 import yaml
```

Comment:
This diff got overly complicated by comparing against the first alternative.  The relevant differences from the original are lines 7-13, 39, and 130-136